website/content/docs/intro/usecases/what-is-a-service-mesh.mdx

```diff
@@ -0,0 +1,112 @@
+
+## How does a Service Mesh work?
+
+A service mesh typically consist of a control plane, and a data plane. The control plane maintains a central registry that keeps track of all services and their respective IP addresses, this is called [service discovery](https://www.hashicorp.com/products/consul/service-discovery-and-health-checking).
+As long as the application is registered with the control plane, the control plane will be able to share with other members of the mesh how to communicate with the application and enforce rules for who can communicate with each other.
+
+The control plane is responsible for securing the mesh, facilitating service discovery, health checking, policy enforcement, and other similar operational concerns.
+
+The data plane handles communication between services.
+Many service mesh solutions employ a sidecar proxy to handle data plane communications, and thus limit the level of awareness the services need to have about the network environment.
+
+![Overview of a service mesh](/img/what_is_service_mesh_1.png)
+
+## API Gateway vs Service Mesh
+
+An API gateway is a centralized access point for handling incoming client requests and delivering them to services.
+The API Gateway acts as a control plane that allows operators and developers to manage incoming client requests and apply different handling logic depending on the request.
+The API Gateway will route the incoming requests to the respective service. API Gateways primary function is to handle requests and return the reply from the service back to the client.
+
+A service mesh specializes in the network management of services and the communication between services.
+The mesh is responsible for keeping track of services and their health status, IP address, traffic routing, and ensuring all the traffic between services are authenticated and encrypted.
+Unlike API Gateways, a service mesh will track all registered services' lifecycle and ensure requests are routed to healthy instances of the service.
+API Gateways are frequently deployed alongside a load balancer to ensure traffic is directed to healthy and available instances of the service.
+The mesh reduces the load balancer footprint as routing responsibilities are handled in a decentralized manner.
+
+API Gateways can be used together with a service mesh to bridge external networks (non-mesh) with a service mesh.
+
+-> **Note**: API Gateways are frequently used to accept north-south based traffic. North-south traffic is networking traffic that either enters or exits a data center or a virtual private network (VPC).
+A service mesh is primarily used for handling east-west based traffic. East-west traffic traditionally remains inside a data center or a VPC.
+A service mesh can be connected to another service mesh in another data center or VPC to form a federated mesh.
```

Comment:
```suggestion
-> **API gateways and traffic direction**: API gateways are often used to accept north-south traffic. North-south traffic is networking traffic that either enters or exits a data center or a virtual private network (VPC).
A service mesh is primarily used for handling east-west traffic. East-west traffic traditionally remains inside a data center or a VPC.
A service mesh can be connected to another service mesh in another data center or VPC to form a federated mesh.
```
Trying to make a more meaningful label for the callout. I also removed "based" because in order to properly punctuate, you would need to write "north-south-based",  which is kind of weird.